articles/security/bulletins/2019-09-05_scopes.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Auth0 Security Bulletin for Assigning Scopes Based on Email Address
+description: Cases where improper custom rule code may create vulnerabilities in the authentication flow.
+topics:
+  - security
+  - security-bulletins
+  - rules
+contentType:
+  - reference
+useCase:
+  - development
+---
+
+# Auth0 Security Bulletin for Assigning Scopes Based on Email Address
+
+## Overview
+
+If you:
+
+* Use rules to assign scopes to users based on their email addresses
```

Comment:
```suggestion
* Use rules to assign scopes to users based on their email addresses **and**
```

Reply:
Only change here, @jeffreylees so that people understand it's not either/or -- it's both; what do you think?